packages/e2e-playwright/models/base.ts

```diff
@@ -14,6 +14,10 @@ export abstract class Base {
         await this.isComponentVisible();
     }
 
+    async isPayable() {
```

Comment:
We have several places where we either say:
`await page.waitForFunction(() => globalThis.component.isValid === true);`
or
`await page.waitForFunction(() => globalThis.component.isValid === false);`

This function could take a parameter & then be used in all these places

Reply:
Yes it's a nice find. I'll refactor this in another PR 😄.